styles/base/scaffolding.css

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: var(--diamond-skin-background);
+  color: var(--diamond-skin-color);
+  margin: 0;
+  min-width: 20em;
```

Comment:
Should this be a var? What's it here to solve?

Reply:
AA support is down to 320px width and this stops the layout getting over squashed, but it's not essential so have removed.